### PR TITLE
Update README to include Audio and High dimensional and screenshot, r…

### PR DESCRIPTION
**Pull request by nickyfantasy**
…ecorder tab order to have better arrangements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +51,23 @@ Image can be used to visualize any tensor or intermediate generated image.
 <img src="https://raw.githubusercontent.com/daming-lu/large_files/master/loss_image.gif" width="60%"/>
 </p>
 
-### Histogram
-Histogram can be used to visualize parameter distribution and trends for any tensor.
+### Audio
+Audio can be used to play input audio samples or generated audio samples.
```
Comment (daming-lu) [inline README.md:55]: generate

File: README.md [hunk 2]
```diff
@@ -52,11 +51,23 @@ Image can be used to visualize any tensor or intermediate generated image.
...
+Cooperated with Python SDK, VisualDL can be compatible with most major DNN frameworks, including
+PaddlePaddle, PyTorch and MXNet.
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/daming-lu/large_files/master/histogram.gif" width="60%"/>
+  <img src="https://raw.githubusercontent.com/daming-lu/large_files/master/graph_demo.gif" width="60%" />
+</p>
+
+### High Dimensional
+High Dimensional can be used to visualize data embeddings by projects high-dimensional data into 2D / 3D.
```
Comment (daming-lu) [inline README.md]: by projecting

**Review by daming-lu — DISMISSED** [review]
Tiny syntax issues

File: README.md (hunk 1 above)
Comment (nickyfantasy) [reply README.md:55]: This sentence means play 'generated' audio samples

**Review by daming-lu — APPROVED** [review]

**Pull request merged**